source/backend/api/Services/DispositionFileService.cs

```diff
@@ -318,6 +370,15 @@ private static void ValidateDispositionOfferStatus(PimsDispositionFile dispositi
             }
         }
 
+        private void ValidateMinistryRegion(long dispositionFileId, short updatedRegion)
+        {
+            short currentRegion = _dispositionFileRepository.GetRegion(dispositionFileId);
+            if (currentRegion != updatedRegion)
+            {
+                throw new UserOverrideException(UserOverrideCode.UpdateRegion, "The Ministry region has been changed, this will result in a change to the file's prefix. This requires user confirmation.");
+            }
+        }
```

Comment:
just confirming this is an actual requirement for disposition files. I did not see this business rule called out in the ACs of PSP-7242